src/main/java/teammates/ui/webapi/request/BasicRequest.java

```diff
@@ -3,7 +3,11 @@
 import teammates.common.exception.InvalidHttpRequestBodyException;
 
 /**
- * The request body of a HTTP request.
+ * A BasicRequest represents the request body of a HTTP request. Within the HTTP request, it is expected that the
```

Comment:
It looks like you are talking about `how` to use it?

And also please avoid talking things in frontend like `web/types/api-output.ts`

Reply:
Yes. Kind of explaining how it is utilized. Because when I was looking through it it was very confusing to pierce everything together. 

How should I refer to it then? Just api-output.ts?

Reply:
Do not make any reference to it at all. The web API stands on its own, with or without `api-output.ts`.